src/api.js

```diff
@@ -529,6 +624,14 @@ var handle = function(handler, context, name) {
   return function(req, res) {
     Promise.resolve(null).then(function() {
       return handler.call(context, req, res);
+    }).then(function() {
+      if (!req.hasAuthed) {
+        // Note: This will not fail the request since a response has already
+        // been sent at this point. It will report to sentry however!
+        // This is only to catch the case where people do not use res.reply()
+        let err = new Error('Deferred auth was never checked and res.reply() not used!');
```

Comment:
`"req.authorized was never called, or some parameters were missing from the request"`